### ISSUE TITLE
[a11y]: Open Notifications Tray Disclosure Button

### ISSUE THREAD

**Issue by peterkulko** [openedx/frontend-app-learning#1880]
## Description

The open notifications tray button lacks mark up and keyboard interaction to support accessibility for someone who uses a screen reader. The following issues are present:

- Pressing tab or the arrow keys does not focus the notification tray after activating the open notifications tray button. It focuses the “Previous” button in the course top bar instead.
- The notifications tray button does not express whether it is expanded or collapsed. 

A person using a screen reader would not know whether the tray was expanded or collapsed, and they would have significant difficulty navigating to the tray once expanded. In smaller viewports a person using keyboard only navigation would be unable to close the notification tray without first tabbing through the background content of the page. 

PR: https://github.com/openedx/frontend-app-learning/pull/1817
